gui-tk/wiring.tcl

```diff
@@ -629,7 +629,7 @@ proc contextMenu {x y X Y} {
                 renameVariableInstances
             }
             .wiring.context add command -label "Edit" -command "editItem"
-            .wiring.context add command -label "Copy" -command "canvas.copyItem"
+            .wiring.context add command -label "Copy" -command "minsky.copy"
```

Comment:
You've change the semantics here. In an item context menu, copy mean "copy this item", and you;ve changed it to mean "copy the selection". Rather confusing, I would think.